app/src/assets/localization/en/pipette_wizard_flows.json

```
@@ -1,15 +1,17 @@
 {
   "before_you_begin": "Before you begin",
-  "remove_labware_to_get_started": "<block>To get started, remove labware from the rest of the deck and clean up the work area to make attachment and calibration easier. Also gather the needed equipment shown on the right hand side</block><block>The calibration probe is included with the robot and should be stored on the right hand side of the door opening.</block>",
-  "attach_probe": "Attach Calibration Probe",
-  "install_probe": "Take the calibration probe from its storage location. Make sure its latch is in the unlocked (straight) position. Press the probe firmly onto the A1 (back left corner) pipette nozzle and then lock the latch. Then test that the probe is securely attached by gently pulling it back and forth.",
+  "remove_labware_to_get_started": "<block>To get started, remove labware from the deck and clean up the working area to make calibration easier. Also gather the needed equipment shown to the right.</block><block>The calibration probe is included with the robot and should be stored on front pillar of the robot.</block>",
```

my bad, this typo was in Figma. fixed there too.
```suggestion
  "remove_labware_to_get_started": "<block>To get started, remove labware from the deck and clean up the working area to make calibration easier. Also gather the needed equipment shown to the right.</block><block>The calibration probe is included with the robot and should be stored on the front pillar of the robot.</block>",
```